services/management/grpc/actions_server.go

```diff
@@ -294,6 +295,60 @@ func (s *actionsServer) StartPTSummaryAction(ctx context.Context, req *managemen
 	}, nil
 }
 
+func pointerToAgentType(agentType models.AgentType) *models.AgentType {
+	return &agentType
+}
+
+// StartPTMySQLSummaryAction starts pt-mysql-summary action and returns the pointer to the response message
+//nolint:lll
+func (s *actionsServer) StartPTMySQLSummaryAction(ctx context.Context, req *managementpb.StartPTMySQLSummaryActionRequest) (*managementpb.StartPTMySQLSummaryActionResponse, error) {
+	// Need to get the service id's pointer to retrieve the list of agent pointers therefrom
+	// to get the particular agentID from the request.
+	service, err := models.FindServiceByID(s.db.Querier, req.ServiceId)
+	if err != nil {
+		return nil, err
+	}
+
+	agents, err := models.FindPMMAgentsRunningOnNode(s.db.Querier, service.NodeID)
+	if err != nil {
+		return nil, status.Errorf(codes.NotFound, "No pmm-agent running on this node")
+	}
+
+	agentID, err := models.FindPmmAgentIDToRunAction(req.PmmAgentId, agents)
+	if err != nil {
+		return nil, err
+	}
+
+	res, err := models.CreateActionResult(s.db.Querier, agentID)
+	if err != nil {
+		return nil, err
+	}
+
+	// Need to get the mysql exporters to get the username and password therefrom
+	psExporters, err := models.FindAgents(s.db.Querier, models.AgentFilters{AgentType: pointerToAgentType(models.MySQLdExporterType)})
+	if err != nil {
+		return nil, err
+	}
+
+	// No exporters found
+	if len(psExporters) == 0 {
+		return nil, status.Errorf(codes.NotFound, "No mysql exporter")
+	}
+
+	// The first exporter's pointer will be used to retrieve the username and password
+	psMySQLExporter := psExporters[0]
```

Comment:
This logic different from what we have for other actions. 
If we have more than one MySQLExporter there we return error.